### PR TITLE
feat - add api 5x in menu

### PR DESCRIPTION
**Pull request by carlosstenzel**

### CODE REVIEW

**Comment by bjohansebas** [comment]
It might be a good idea to add it in all languages, even if it's not translated into their language yet.

File: _includes/header/header-de.html [hunk 1]
```diff
@@ -77,6 +77,8 @@
               <ul id="application-menu" class="menu">
                   <li><a href="/{{ page.lang }}/4x/api.html"{% if page.menu == 'api' %} class="active"{% endif %}>API-Referenz</a>
                       <ul>
+                          <li><a href="/{{ page.lang }}/5x/api.html">5.x (beta)</a>
```
Comment (bjohansebas) [inline _includes/header/header-de.html]:
```suggestion
                          <li><a href="/{{ page.lang }}/5x/api.html">5.x</a>
```

**Review by bjohansebas — COMMENTED** [review]
The (beta) label has already been removed, it would be better to eliminate it from all headers

**Comment by crandmck** [comment]
Yes, per https://github.com/expressjs/expressjs.com/pull/1603, v5 is no longer in beta.  Please remove that from all langs.

**Comment by carlosstenzel** [comment]
@bjohansebas @crandmck 
I removed the 'beta' text

**Review by bjohansebas — APPROVED** [review]
LGTM

**Review by crandmck — APPROVED** [review]

**Pull request merged**